README.md

```diff
@@ -0,0 +1,156 @@
+
+## 🎯 Purpose
+
+This repository serves as a benchmark and learning resource for epidemiological modeling, featuring:
+
+- **Simple, reproducible examples** of common epidemiological models
+- **Cross-language implementations** in both R and Python
+- **Consistent documentation** using Quarto documents
+- **Performance benchmarking** across different modeling frameworks
+- **Best practices** for epidemiological ABM development
+
+## 🚀 Getting Started
+
+### Prerequisites
+
+This repository uses a devcontainer environment for consistent development across different systems. You'll need:
+
+- [Docker](https://www.docker.com/get-started)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+
+### Using the Devcontainer
+
+1. Clone this repository:
+   ```bash
+   git clone https://github.com/EpiForeSITE/abm-benchmark.git
+   cd abm-benchmark
+   ```
+
+2. Open in VS Code:
+   ```bash
+   code .
+   ```
+
+3. When prompted, click "Reopen in Container" or use the command palette (Ctrl/Cmd + Shift + P) and select "Dev Containers: Reopen in Container"
+
+4. The devcontainer will automatically:
+   - Set up the R and Python environments
+   - Install necessary packages and dependencies
+   - Configure VS Code extensions for R, Python, and Quarto
+
+### Environment Details
+
+The devcontainer is based on `rocker/tidyverse:4.5.0` and includes:
+
+- **R environment**: Pre-configured with tidyverse and common epidemiological packages
+- **Python environment**: Managed with `uv`, includes NumPy, pandas, matplotlib, and scientific computing libraries
+- **Quarto**: For creating reproducible documents combining code and narrative
+- **RStudio Server**: Available on port 8787 for R development
+
+## 📁 Repository Structure
+
+Examples are organized in numbered folders following this structure:
+
+```
+XX-modeltype_package/
+├── README.qmd          # Quarto document with the example
+├── README.md          # Rendered markdown (auto-generated)
+└── data/              # Input data (if needed)
+```
+
+### Current Examples
+
+*Examples will be added here as they are contributed to the repository.*
+
+## 📝 Example Structure
+
+Each example follows a consistent format:
+
+1. **Metadata**: Date, author, related PR/issue
+2. **Introduction**: Brief overview of the model and its applications
+3. **Model Description**: 
+   - Mathematical formulation
+   - Model type (network-based vs. perfect mixing)
+   - Time structure (continuous vs. discrete)
+4. **Implementation**: 
+   - Code execution with timing information
+   - Visualization of epidemiological curves
+   - Performance metrics
+5. **References**: Links to original implementations, packages, and literature
+
+## 🤝 How to Contribute
```

Comment:
Let's move this entire section to a "CONTRIBUTE.md" document. The "How to contribute" title can still be here, but it should be brief and point directly to the "CONTRIBUTE.md" document.